### DISCUSSION TITLE
Grow to size.

### DISCUSSION THREAD

**Discussion by RanarTheWrathful** [Taureon/aps-plus-plus#215] [Suggestions]
This is something for your projectile definitions, so instead of having an oversized barrel or having a small Barrel shooting something huge, you can do GROW_TO_SIZE: true, it will shoot at the size of the barrel width then grow to its true size, similar to when something levels from score.

**Comment by Trioplane** [comment]
so its just, `MOTION_TYPE; 'grow'` but with a limit

**Comment by Taureon** [comment]
> so its just, `MOTION_TYPE; 'grow'` but with a limit

lol
"growth" as a motion type is mid
its a motion type not a size change type
